package.json

```diff
@@ -1,16 +1,58 @@
 {
     "name": "@superfluid-finance/protocol-monorepo",
     "description": "Superfluid Protocol monorepo",
-    "repository": {
-        "type": "git",
-        "url": "github:superfluid-finance/protocol-monorepo"
+    "version": "0.0.0",
+    "devDependencies": {
+        "@nomicfoundation/hardhat-chai-matchers": "1.0.6",
+        "@nomicfoundation/hardhat-ethers": "3.0.7",
+        "@nomiclabs/hardhat-etherscan": "^3.1.7",
+        "@nomiclabs/hardhat-web3": "^2.0.0",
+        "@openzeppelin/test-helpers": "^0.5.16",
+        "@truffle/hdwallet-provider": "^2.1.15",
+        "@typechain/ethers-v5": "^11.1.2",
+        "@typechain/hardhat": "9.1.0",
+        "@types/lodash": "^4.17.7",
+        "@types/mocha": "^10.0.7",
+        "@typescript-eslint/eslint-plugin": "8.3.0",
+        "@typescript-eslint/parser": "8.3.0",
```

Comment:
Not sure if some of these locked versions are on purpose

Reply:
probably not, not sure why it happenned with syncpack updating them

Reply:
Codified the rules with the syncpack:

```
semverGroups:
  # Note: rule priority is top (high) to bottom (low)
  - label: Use looser ranges for metadata and ethereum-contracts local packages
    dependencies:
      - "@superfluid-finance/metadata"
      - "@superfluid-finance/ethereum-contracts"
    range: "^"
  - label: Use exact version for prod dependencies
    dependencyTypes: ["prod"]
    range: ""
  - label: Use looser range by default
    range: "^"

```